Adding the Pledge & Champions Program

mgifford opened the pull request:
Putting in references in our handbook to these two initiatives.

[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/mgifford-patch-2/ ![Documentation Status](https://readthedocs.org/projects/civicactions-handbook/badge/?version=mgifford-patch-2)

[//]: # (rtdbot-end)

mgifford committed:
Adding the Pledge & Champions Program

Putting in references in our handbook to these two initiatives.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra reviewed (approved):
Thanks @mgifford. Looking good!

mgifford committed:
Just simplifying the page

This section is repeated in the practice area so just simplifying it.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

mgifford committed:
Realized I dropped the champions program 

I had this earlier in the culture page, but then wanted to consolidate the "how you can join" sections.

dmundra commented on docs/020-about-us/culture.md:
```diff
@@ -98,11 +100,9 @@ CivicActions Accessibility is an open, agile project, practice area and communit
-- Join the #accessibility Slack channel
-- [Submit an "Add new team member" issue](https://github.com/CivicActions/accessibility/issues/new/choose) : [Join GitHub](https://github.com/) if you haven't already
-- [Join the meetings](https://accessibility.civicactions.com/calendar) check-ins + prioritization + practice area calls
+Check out our [Accessibility Practice Area](https://handbook.civicactions.com/en/latest/050-how-we-work/practice-areas/accessibility-practice-area/)!
```
@mgifford can you make this link relative?

mgifford replied:
Sure thing. Good catch.

dmundra reviewed (changes requested):
Minor change.

mgifford committed:
Everything should be relative

Especially relative links.

mgifford committed:
Relative to GitHub

Adjusting. Thanks @dmundra I thought that was not quite right.

mgifford committed:
What does he mean by latest

Oops..  Right, versions..  Thanks!

dmundra committed:
Merge branch 'master' into mgifford-patch-2

dmundra committed:
Fixing links

dmundra committed:
Fixing links one more time.

dmundra commented on docs/020-about-us/culture.md:
```diff
@@ -98,11 +100,9 @@ CivicActions Accessibility is an open, agile project, practice area and communit
-- Join the #accessibility Slack channel
-- [Submit an "Add new team member" issue](https://github.com/CivicActions/accessibility/issues/new/choose) : [Join GitHub](https://github.com/) if you haven't already
-- [Join the meetings](https://accessibility.civicactions.com/calendar) check-ins + prioritization + practice area calls
+Check out our [Accessibility Practice Area](../../050-how-we-work/practice-areas/accessibility-practice-area)!
```
@grugnog remark-validate-links is complaining about this path but I am not sure what the write value is here when it gets converted to read the docs (see URL https://handbook.civicactions.com/en/mgifford-patch-2/020-about-us/culture/). I tried doing only one directory traversal in the previous commit but I still got a complaint. Any ideas?

dmundra replied:
When I did the previous commit the links in preview site https://handbook.civicactions.com/en/mgifford-patch-2/020-about-us/culture/ did work. Right now they are 404s.

dmundra committed:
Update culture.md
